### ISSUE TITLE
Create Documentation Website

### ISSUE THREAD

**Issue by William-McGonagle** [William-McGonagle/burm#6]

**Comment by adetyaz** [comment]
Hello, first timer here, can I work on this? Also is there an available design to follow

**Comment by adetyaz** [comment]
Alright no problem, I don't have experience with working with CI/CD, I'll find a way to figure that out. While I'll be trying to figure that out, I'll start on the website creation taking inspiration from other documentation websites. Anyways I'll still be asking a few questions.

**Comment by namanthanki** [comment]
I can help you with a starter web design created in figma and see it if you like it, I have never contributed on github so I would love to contribute and help in this early stage, can you provide extra information on how would you like to have your landing page designed?

**Comment by adetyaz** [comment]
No problem with that, I'll appreciate that, it'll be easier when I have something to follow. @namanthanki

**Comment by William-McGonagle** [comment]
Hey @namanthanki- If you want to build a landing page or documentation page, just theme it similar to how [Bun](https://bun.sh/) is designed. Thanks for helping!

**Comment by namanthanki** [comment]
@William-McGonagle Hi, I tried creating first prototype of the landing page and I kept it single section, single page philosophy with thought that current needs of documentation can be showcased through github readme however in future we might need a separate documentation page but as of now I have tried my best to keep it simple and stupid, with your recommendation in consideration I have tried to make it similar to bun site but with touch of its own color scheme, let me know what you think about it. If you approve this page I can turn it into real working page and share it with you in no time 
Figma Link (If you want to play around): https://www.figma.com/file/ob0d2DLDifcEk28jmMUuti/Burm?node-id=0%3A1
![Burm](https://user-images.githubusercontent.com/49157253/179820008-2b99fcb4-aa99-4804-9bfb-b4c6dcfc7c28.png)

**Comment by adetyaz** [comment]
I think this looks good @namanthanki 

@William-McGonagle does it matter if i build the website using plain html, css, js or nextjs.

**Comment by William-McGonagle** [comment]
I think the design looks great @namanthanki. I think that we should change the color from green to something more doughy and bun-like though- what are everyone's thoughts on that? 

Also @adetyaz, you can build it however you would like. It just has to be able to deploy to Github pages and preferably works with CI/CD. 

Sidenote- *This is just an idea, but wouldn't it be cool if you queried data from the [NPM API](https://registry.npmjs.com/burm/latest), so that the documentation system could be switched in for other projects.*

**Comment by namanthanki** [comment]
@William-McGonagle With your consideration I have tried my best to come up with a color palette which is more doughy themed as you mentioned, let me know your thoughts about this 
![Burm_v2](https://user-images.githubusercontent.com/49157253/179968424-2174beaf-cd1c-4fbd-ac70-a8d363eb41d3.png)

**Comment by William-McGonagle** [comment]
@namanthanki I think it looks great! Thank you so much for helping with the project- do you also think the documentation needs a design, or do you think that we should just use a template?

**Comment by namanthanki** [comment]
@William-McGonagle Yes documentation page could be designed as well, as of now I am trying to look for inspiration to create a minimalistic documentation page, I'll let you know if I create any prototypes.

**Comment by William-McGonagle** [comment]
@namanthanki Sounds great!

**Comment by adetyaz** [comment]
@namanthanki is the recent design the one that'll be used or you have plans on making changes to it?

**Comment by namanthanki** [comment]
@adetyaz that's the one, it's the landing page we're going to use as of now. If you need any help in designing or writing code let me know
